src/networking/torii.rs

```diff
@@ -10,6 +11,12 @@ use std::{
     time::{Duration, Instant},
 };
 
+const TORII_URL: &str = "127.0.0.1:1337";
```

Comment:
This would be better read in from a config file

Reply:
https://soramitsu.atlassian.net/browse/RH2-33